### DISCUSSION TITLE
Suggested Solution for "Zotero missing from Word"

### DISCUSSION THREAD

**Discussion by JuliusBairaktaris** [HotCakeX/Harden-Windows-Security#90] [General]
I believe it would be beneficial for the Wiki to include a dedicated section addressing these common problems and their solutions.

For example:

### Suggested Solution for "Zotero missing from Word":

1. Open the Windows Registry Editor (regedit).
2. Navigate to: `Computer\HKEY_CURRENT_USER\Software\Policies\Microsoft\office\16.0\common\toolbars\word\noextensibilitycustomizationfromdocument` Change the value from `1` to `0`.
3. Next, navigate to: `Computer\HKEY_CURRENT_USER\Software\Policies\Microsoft\office\16.0\word\security\vbawarnings` Change the value from `3` to `2`.

By making these changes, Zotero should appear in Word as expected.

It would be fantastic if this solution could be added to the Wiki and as a script category, as it might save users a lot of time searching for fixes to this common problem. If there are other common problems related to the security script, it might be worth considering adding them to this section as well.

**Comment by JuliusBairaktaris** [comment]
The values change back after every reboot....

@HotCakeX do you have an idea?

**Comment by HotCakeX** [comment]
Hi,
I'm not familiar with what that registry key does, sorry
the wiki posts aren't indexed by search engines that's why I put them in a folder in the root of the repo and also put them on the website https://spynetgirl.github.io/

the key is in the HKEY_CURRENT_USER hive and it only affects the current user. the reason it's changing after reboot might be because you already applied Microsoft 365 apps baselines and this registry key is overriding one of those settings and since group policies take priority your manual customization disappears.

**Comment by JuliusBairaktaris** [comment]
I attempted to locate the group policies related to macros but was unsuccessful. Consequently, I opted to modify the registry keys.

**Comment by HotCakeX** [comment]
> I attempted to locate the group policies related to macros but was unsuccessful. Consequently, I opted to modify the registry keys.

There are Word experts here in this Microsoft Tech community space if you want to ask your question there
https://techcommunity.microsoft.com/t5/word/bd-p/Word

**Comment by HotCakeX** [comment]
So to wrap up this issue:

> If there are other common problems related to the security script, it might be worth considering adding them to this section as well.

Any problem with the script or other tools in the repository will be fixed as soon as possible, if you notice anything please open a new issue.

I did some research, Zotero is a plugin for Word, almost all of its features are available in latest M365 Word.
You provided this solution to fixing a problem, so in order for anyone else to find it easier, I'm going to convert this issue to a discussion and change its title to match its purpose, it will be there and people will find it.
